app/code/Dss/Limitcartqty/etc/adminhtml/system.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0"?>
+<!-- /**
+ * Digit Software Solutions..
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the EULA
+ * that is bundled with this package in the file LICENSE.txt.
+ *
+ * @category   Dss
+ * @package    Dss_Limitcartqty
+ * @author     Extension Team
+ * @copyright Copyright (c) 2024 Digit Software Solutions. ( https://digitsoftsol.com )
+ */ -->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="../../../Config/etc/system_file.xsd">
+    <system>
+        <tab id="bss" translate="label" sortOrder="300">
+            <label><![CDATA[Dss Commerce]]></label>
+        </tab>
+        <section id="Dss_Commerce" translate="label" type="text" sortOrder="100" showInDefault="1"
+            showInWebsite="1" showInStore="1">
+            <label>LimitCartQty</label>
+            <tab>bss</tab>
+            <resource>Dss_Limitcartqty::config</resource>
+            <group id="item_options" translate="label" type="text" sortOrder="50" showInDefault="1"
+                showInWebsite="1" showInStore="1">
+                <label>Product Stock Options</label>
+                <field id="Dss_min_total_qty" translate="label" sortOrder="20" showInDefault="1"
```

Comment:
use proper name for id and name

Reply:
@harishmk1805 
use proper name for id and name added.